spec/models/mention_spec.rb

```diff
@@ -30,14 +30,18 @@
 
   describe '#notify_later' do
     it 'should queue the notification' do
-      mention = build :mention, mentionable: focus
+      project_board = create :board, section: "project-#{ focus.project.id}"
+      project_discussion = create :discussion, board: project_board
+      project_comment = create :comment, discussion: project_discussion
+      mention = build :mention, mentionable: focus, comment: project_comment
       expect(MentionWorker).to receive(:perform_async)
       mention.save!
     end
   end
 
   describe '#notify_mentioned' do
     it 'should notify of the mention' do
+      subject_mentioned =
```

Comment:
I think you may have forgotten this. But for that, all looks good

Reply:
Woops! good catch. i must have accidentally typed this when i was debugging